### PR TITLE
Update dca full names

### PR DESCRIPTION
**Pull request by rawOrlando**
Updates DCA names and variations shown in https://tuftswork.atlassian.net/jira/software/c/projects/TDLR/boards/58?modal=detail&selectedIssue=TDLR-2452.

### CODE REVIEW

**Comment by rawOrlando** [comment]
@mkorcy one more small split off piece of dca name change for Mira

**Review by mkorcy — COMMENTED** [review]
I'm not sure about that one change in the draft controller here.

File: spec/controllers/tufts/draft_controller_spec.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
...
...> [""], "qr_status" => [""], "rejection_reason" => [""], "replaces" => [""], "resource_type" => [""], "retention_period" => [""], "rights_holder" => [""], "rights_note" => "", "geographic_name" => [""], "steward" => "dca", "subject" => [""], "table_of_contents" => [""], "temporal" => [""], "is_part_of" => [""], "rights_statement" => 'http://tarc.tufts.edu/ua/access/rights-creator.html', "license" => ['http://tarc.tufts.edu/ua/access/rights-creator.html'], "admin_set_id" => "admin_set/default", "member_of_collection_ids" => [""], "permissions_attributes" => { "1" => { "access" => "edit", "id" => "943d67c1-ed47-48d0-bd0f-7476610593bb/11/82/8f/f2/11828ff2-9979-4d95-a742-5ae1fb08f37d" } }, "visibility" => "open", "visibility_during_embargo" => "restricted", "embargo_release_date" => "2017-11-14", "visibility_after_embargo" => "open", "visibility_during_lease" => "open", "lease_expiration_date" => "2017-11-14", "visibility_after_lease" => "restricted", "version" => "W/\"60c82ca9e6797d107b8b21e8cb983290863de8a1\"" }, "find_child_work" => "", "new_user_name_skel" => "", "new_user_permission_skel" => "none", "new_group_name_skel" => "Select a group", "new_group_permission_skel" => "none" }
```
[the remark this answers is not in the document]
Comment (rawOrlando) [reply spec/controllers/tufts/draft_controller_spec.rb:17]: I can pull this out. But I expect to have to rebase for this line.

[the remark this answers is not in the document]
Comment (mkorcy) [reply spec/controllers/tufts/draft_controller_spec.rb:17]: that's fine if you want to rebase.  it's a mess of a line was just trying to make it easy for you.

**Pull request merged**